sklearn/model_selection/_search.py

```diff
@@ -867,6 +870,9 @@ def _store(key_name, array, weights=None, splits=False, rank=False):
 
         return results
 
+    def _more_tags(self):
+        return self.estimator._get_tags()
```

Comment:
Very interesting.

Reply:
This is the same approach than `sklearn.feature_selection.RFE`, that seems to work pretty well :)